slices/apache2-bin.yaml

```diff
@@ -0,0 +1,1039 @@
+package: apache2-bin
+
+essential:
+  - apache2-bin_copyright
+
+slices:
+  apache2:
+    essential:
+      - libapr1t64_libs
+      - libaprutil1t64_libs
+      - libpcre2-8-0_libs
+    contents:
+      /usr/sbin/apache2:
+
+  bins:
+    essential:
+      - apache2-bin_apache2
+      - apache2-bin_modules
+      - apache2-bin_state
+      - dash_bins
```

Comment:
I don't think the `apache2-bin` stuff requires `dash_bins`, since there are no scripts?
Therefore I think we can omit it?